create.js

```diff
@@ -0,0 +1,90 @@
+const {MongoClient} = require('mongodb');
+
+async function main(){
+    /**
+     * Connection URI. Update <username>, <password>, and <your-cluster-url> to reflect your cluster.
+     * See http://bit.ly/NodeDocs_lauren for more details
+     */
+    const uri = "mongodb+srv://<username>:<password>@<your-cluster-url>/test?retryWrites=true&w=majority";
+
+    /**
+     * The Mongo Client you will use to interact with your database
+     * See bit.ly/Node_MongoClient for more details
+     */
+    const client = new MongoClient(uri);
+
+    try {
+        // Connect to the MongoDB cluster
+        await client.connect();
+
+        // Make the appropriate DB calls
+
+        // Create a single new listing
+        await createListing(client,
+            {
+                name: "Lovely Loft",
+                summary: "A charming loft in Paris",
+                bedrooms: 1,
+                bathrooms: 1
+            }
+        );
+  
+        // Create 3 new listings
+        await createMultipleListings(client, [
+            {
+                name: "Infinite Views",
+                summary: "Modern home with infinite views from the infinity pool",
+                property_type: "House",
+                bedrooms: 5,
+                bathrooms: 4.5,
+                beds: 5
+            },
+            {
+                name: "Private room in London",
+                property_type: "Apartment",
+                bedrooms: 1,
+                bathroom: 1
+            },
+            {
+                name: "Beautiful Beach House",
+                summary: "Enjoy relaxed beach living in this house with a private beach",
+                bedrooms: 4,
+                bathrooms: 2.5,
+                beds: 7,
+                last_review: new Date()
+            }
+        ]);
+
+    } catch (e) {
+        console.error(e);
```

Comment:
Minor point: this `catch` block will log any error that occurs in the above `try` block, but it will also stop the error from propagating. Since you already have a `.catch` on the `main` function invocation at line 66, this might not be necessary.

That said, if you think this is clearer for readers on how to catch the errors, then it is fine.